client/containers/App/index.js

```diff
@@ -7,15 +7,15 @@ import Reports from 'containers/Reports';
 import './style.scss';
 
 const App = () => (
-  <div>
+  <main>
```

Comment:
`main` is a html5 tag. IE9+ native support or can be `thought` for IE8. Since React supports IE9+, we should be alright